tests/lapi/debug_getlocal_test.lua

```diff
@@ -0,0 +1,67 @@
+--[[
+SPDX-License-Identifier: ISC
+Copyright (c) 2023-2025, Sergey Bronnikov.
+
+Negation overflow in getlocal/setlocal,
+https://github.com/lua/lua/commit/a585eae6e7ada1ca9271607a4f48dfb17868ab7b
+
+debug.getlocal on a coroutine suspended in a hook can crash the interpreter,
+https://www.lua.org/bugs.html#5.3.0-2
+
+Synopsis: debug.getlocal([thread,] level, local)
+]]
+
+local luzer = require("luzer")
+local test_lib = require("lib")
+
+-- Extract locals.
+local function debug_hook()
+    local level = 2
+    local i = 1
+    while true do
```

Comment:
We may also scan all levels to the lowest one.

Reply:
Updated, now all levels are scanned.